lib/cog_api/fake/relay_groups.ex

```diff
@@ -28,6 +32,14 @@ defmodule CogApi.Fake.RelayGroups do
   end
 
   def delete(_, %Endpoint{token: nil}), do: Endpoint.invalid_endpoint
+  def delete(%{name: name}, %Endpoint{}) do
+    relay_group = Server.show_by_key(RelayGroup, :name, name)
+    if relay_group do
```

Comment:
This conditional seems to duplicate the functionality of the other `delete` function. Could we pull out a shared method between the two that just takes in a `relay_group`?